inst/components/scss/sidebar.scss

```diff
@@ -72,8 +74,6 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
 
   > .sidebar {
     grid-column: 1 / 2;
-    width: 100%;
-    height: 100%;
     border-right: var(--_vert-border);
```

Comment:
I think we should bring back the `width: 100%`

Reply:
I checked all of our rules and `.sidebar` always has both `grid-column` and `grid-row` properties, which drive the height and width of the sidebar container. I'm comfortable with both being redundant, but happy to leave `width` for now if you want.